src/main/java/com/google/step/snippet/external/StackOverflowClient.java

```diff
@@ -73,8 +69,8 @@ public Card search(String url) {
       return null;
     }
     // No description or code is allowed for StackOverflow.
-    String description = getDescription(answerBody);
-    String code = getCode(answerBody);
+    String description = getAnswerBody(answerId);
+    String code = null;
```

Comment:
Setting this to null leads to `<code></code>` being generated within the card on search.jsp.

There should be a null check included (probably within search.jsp itself, to make sure this empty code tag is not generated when `knowledgeCard.getCode() == null`)

Otherwise this new format looks way really good!

Reply:
dealt with this in this pr but probably also need to be solve in #27 ?

Reply:
I think fixing here should be fine. The null check wasn't required before because we did the null check in the scraping clients themselves and never returned a new card if code was null.